rootston/cursor.c

```diff
@@ -134,7 +134,9 @@ static void roots_cursor_update_position(struct roots_cursor *cursor,
 		} if (view && surface) {
 			// motion over a view surface
 			wlr_seat_pointer_notify_enter(seat->seat, surface, sx, sy);
-			wlr_seat_pointer_notify_motion(seat->seat, time, sx, sy);
+			if (time) {
```

Comment:
I don't really like this, but it feels better than asking for time and notifying for motion that hasn't really happened anyway (we're moving a window, not the pointer).
Might be able to rip part of this function off to share code instead of hijacking it.

Reply:
Yeah, what we're trying to do here is faking input events, and it doesn't work well because we don't know the time reference. I'm still thinking we should send pointer motion events, but I'm not sure how to fix this. We could try to get the time from the seat that triggered the update, but nothing guarantees that all input devices are from the same backend and have the same time reference.

Reply:
I also originally tried that but we don't always have an input event related to the call. Sometimes it's a key event (e.g. alt-tab) and we have one, but when a new window pops in we don't have anything :/

I haven't totally given up on this PR btw but really won't have time for another while, sorry